engine/src/main/scala/cromwell/webservice/routes/CromwellApiService.scala

```diff
@@ -175,6 +179,29 @@ trait CromwellApiService extends HttpInstrumentation with MetadataRouteSupport {
     }
   } ~ metadataRoutes
 
+
+  private def metadataLookupForTimingRoute(workflowId: WorkflowId): Future[MetadataBuilderActorResponse] = {
+    val includeKeys = NonEmptyList.of("start", "end", "executionStatus", "executionEvents", "subWorkflowMetadata")
+    val readMetadataRequest = (w: WorkflowId) => GetSingleWorkflowMetadataAction(w, Option(includeKeys), None, expandSubWorkflows = true)
+
+    metadataBuilderRegulatorActor.ask(readMetadataRequest(workflowId)).mapTo[MetadataBuilderActorResponse]
+  }
+
+  private def completeTimingRouteResponse(metadataResponse: Future[MetadataBuilderActorResponse]) = {
+    onComplete(metadataResponse) {
+      case Success(r: BuiltMetadataResponse) => {
+        Try(Source.fromResource("workflowTimings/workflowTimings.html").mkString) match {
```

Comment:
ToL: I think this HTML is small enough that it could be saved in a string (lazy?) `val`, instead of decompressed from disk on every request.

Reply:
I thought about suggesting this but walked back - I'd hope that `Source.fromResource` is smart enough to handle that kind of caching for us, and hopefully in a smarter way than we would come up with ad-hoc

Reply:
Talked IRL. No reason to over optimize at this time. If measured performance happens to be a problem, we'll fix it then.